looks like we need to remove the framework import to make the tools happy.

``` utils/scheduler/scheduler.go:26:2: "k8s.io/autoscaler/cluster-autoscaler/simulator/framework" imported and not used ```

--- a/cluster-autoscaler/utils/scheduler/scheduler.go
+++ b/cluster-autoscaler/utils/scheduler/scheduler.go
@@ -23,7 +23,6 @@ import (
 
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
-	"k8s.io/autoscaler/cluster-autoscaler/simulator/framework"
 	schedulerinterface "k8s.io/kube-scheduler/framework"
 	scheduler_config "k8s.io/kubernetes/pkg/scheduler/apis/config"
 	scheduler_scheme "k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"

--- a/cluster-autoscaler/utils/scheduler/scheduler_test.go
+++ b/cluster-autoscaler/utils/scheduler/scheduler_test.go
@@ -25,7 +25,6 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/resource"
 	testconfig "k8s.io/autoscaler/cluster-autoscaler/config/test"
-	. "k8s.io/autoscaler/cluster-autoscaler/utils/test"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 	schedulerimpl "k8s.io/kubernetes/pkg/scheduler/framework"
 

